test/spec/FindInFiles-test.js

```diff
@@ -785,6 +785,7 @@ define(function (require, exports, module) {
                 gotChange = false;
                 oldResults = null;
                 wasQuickChange = false;
+                FindInFiles.searchModel.clear();
```

Comment:
Maybe call `FindInFiles.clearSearch`? It is marked as private method but it seems it is already used outside that file.